### PR TITLE
Data release 2023-10-18

### PR DESCRIPTION
**Pull request by juliawu**

### CODE REVIEW

File: internal/server/v2/facet/golden/series_facet/series_facet.json [hunk 1]
```diff
@@ -33,58 +33,66 @@
               ]
             },
             {
-              "facet_id": "217147238",
+              "facet_id": "1151455814",
               "observations": [
                 {
-                  "value": 5
+                  "value": 28
```
Comment (pradh) [inline internal/server/v2/facet/golden/series_facet/series_facet.json:39]: @shifucun what is the value here?

File: internal/server/v1/event/golden/event_collection/CycloneEvent_Earth_201005.json [hunk 2]
```diff
@@ -1,56 +1,6 @@
...
-              "Millibar989"
-            ]
-          },
-          "name": {
-            "vals": [
-              "Bandu"
-            ]
-          }
-        },
-        "dcid": "cyclone/ibtracs_2010136N09057",
```
Comment (pradh) [inline internal/server/v1/event/golden/event_collection/CycloneEvent_Earth_201005.json:51]: @ajaits is this deletion expected?

**Review by pradh — APPROVED** [review]
Couple of questions, otherwise LG.

File: internal/server/v2/facet/golden/series_facet/series_facet.json (hunk 1 above)
Comment (shifucun) [reply internal/server/v2/facet/golden/series_facet/series_facet.json:39]: This is the number of observations for this facet. @chejennifer with the change to add earliest and latest date, I wonder how do we put "earliest_data", "latest_date" and "obs_count" here together that still conforms the V2 observation response.

One option is to have 

`
[{"date": "2010"}, {"date": "2020"} 
`
But count is hard to fill in.

Any ideas?

File: internal/server/v1/event/golden/event_collection/CycloneEvent_Earth_201005.json (hunk 2 above)
Comment (ajaits) [reply internal/server/v1/event/golden/event_collection/CycloneEvent_Earth_201005.json:51]: No, the cyclone events have had no script change. 

Does adding geoJson for ocean basins affect IBTrACS? 

Is there a way to query for all cyclone events that have been deleted to see if only the ones in the oceans that got the oceanBasin boundaries are missing?

Comment (pradh) [reply internal/server/v1/event/golden/event_collection/CycloneEvent_Earth_201005.json:51]: Not sure how adding geoJson for ocean basis affects IBTrACS.  We would need to run the import differ between two caches I think?

File: internal/server/v2/facet/golden/series_facet/series_facet.json (hunk 1 above)
Comment (chejennifer) [reply internal/server/v2/facet/golden/series_facet/series_facet.json:39]: sorry, I missed this, but we've chatted offline about updating the response structure

**Comment by shifucun** [comment]
shall we close this? @juliawu

**Comment by juliawu** [comment]
> shall we close this? @juliawu

Yes. Closing since this PR was meant as a data check in-between releases, and has been superceded by https://github.com/datacommonsorg/mixer/pull/1297

**Pull request closed without merging**